include/open62541/plugin/accesscontrol.h

```diff
@@ -127,6 +137,8 @@ struct UA_AccessControl {
                                                       UA_DateTime endTimestamp,
                                                       bool isDeleteModified);
 #endif
+    UA_Boolean (*checkUserDatabase)(const UA_UserNameIdentityToken *userToken, UA_String *roleName);
```

Comment:
What does this do?
Validate that the user has a certain role?

Reply:
`checkUserDatabase` is not required and is removed.